Documentation/connectors/github.md

```diff
@@ -51,6 +52,15 @@ connectors:
       teams:
       - red-team
       - blue-team
+
+    # Optional choice between 'name' (default), 'slug' or 'both'.
+    #
+    # As an example, group claims for member of 'Site Reliability Engineers' in
+    # Acme organization would yield:
+    #   - ['acme:Site Reliability Engineers'] for 'name'
+    #   - ['acme:site-reliability-engineers'] for 'slug'
+    #   - ['acme:Site Reliability Engineers', 'acme:site-reliability-engineers'] for 'both'
```

Comment:
Do you have a use case for 'both'? I'd rather we drop it if not. If we ever add another field (maybe ID?) 'both' becomes a weird setting.